storybook/src/css/Button.css

```diff
@@ -0,0 +1,33 @@
+.custom-button {
+    border: none;
+    padding: 12px 20px;
+    color: white;
+    text-align: center;
+    text-decoration: none !important;
+    cursor: pointer;
+    display: inline-block;
```

Comment:
You should try and understand what each of these things mean, I would say the display options are one of the most important.